appscale/tools/admin_api/version.py

```diff
@@ -104,6 +104,22 @@ def from_yaml(app_yaml):
         except StandardError:
             raise AppEngineConfigException('Invalid app.yaml: automatic_scaling invalid.')
 
+        # Adds optional elements for automatic scaling.
+        try:
+            min_idle = automatic_scaling.get('min_idle_instances')
+            if min_idle is not None:
+                version.automatic_scaling['minIdleInstances'] = int(min_idle)
+            max_idle = automatic_scaling.get('max_idle_instances')
+            if max_idle is not None:
+                version.automatic_scaling['maxIdleInstances'] = int(max_idle)
+            max_concurrent = automatic_scaling.get('max_concurrent_requests')
+            if max_concurrent is not None:
+                version.automatic_scaling['maxConcurrentRequests'] =
+                                          int(max_concurrent)
```

Comment:
If you span multiple lines during an assignment, you need to either escape the line break or wrap it in parens.